### PR TITLE
travis pls halp

### PR DESCRIPTION
**Pull request by Borgaard**

### CODE REVIEW

File: NicoleBorgaard-Borgaard/hacker_news_onion.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+=begin
```
Comment (Borgaard) [inline NicoleBorgaard-Borgaard/hacker_news_onion.rb]: Ignore this, I just find them hilarious.

**Comment by Borgaard** [comment]
Just wanting confirmation this will work out okay! Or maybe it'll break everything and we'll have a learning experience on our hands. I just want us to have the learning experience _before_ subjecting ~32 students to pulling this upstream.

**Comment by tgaff** [comment]
That will work

can you rm the .DS_Store while you're at it.

**Comment by Borgaard** [comment]
.DS_Store pwnd!

**Comment by tgaff** [comment]
:skull:

**Comment by Borgaard** [comment]
:octopus:

**Pull request merged**